src/icons/languageIcons.ts

```diff
@@ -67,6 +67,7 @@ export const languageIcons: LanguageIcon[] = [
     { icon: { name: 'processing' }, ids: ['processing'] },
     { icon: { name: 'hcl' }, ids: ['hcl'] },
     { icon: { name: 'go' }, ids: ['go'] },
+    { icon: { name: 'go-mod' }, ids: ['go.sum', 'go.mod'] },
```

Comment:
Are the language ids necessary? Language ids != file extensions.
The language ids are necessary to assign an icon to certain languages, not file names. Just asking.

Reply:
Yes. The syntax is different, so I guess the extension creates different languages for each file. For it to work in the language selection menu, the languageIcons should be set as well.

![Screen Shot 2019-06-20 at 08 03 50](https://user-images.githubusercontent.com/34144667/59846762-1ff2c900-9337-11e9-8098-086e9ce745b3.png)

Reply:
Sorry, just did some more testing and the `languageIcons` entry is not actually necessary, the problem was that I was using `fileExtensions` instead of `fileNames` in the `fileIcons` file, changed it and it's working as intended!